attributes/default.rb

```diff
@@ -20,6 +20,8 @@
 default['chatops_deployer']['docker_auto_build']['path'] = '/usr/lib/docker_auto_build'
 # Port on which docker_auto_build app listens
 default['chatops_deployer']['docker_auto_build']['port'] = '8001'
+# Directory where docker_gc will be cloned
+default['chatops_deployer']['docker_gc']['path'] = '/usr/lib/docker_gc'
```

Comment:
Can we replace all `docker_gc` with `docker-gc` since that's the name of the tool ?